### DISCUSSION TITLE
How to perform Junit testing for AmazonS3 client maxRetryCount verification

### DISCUSSION THREAD

**Discussion by atmel21** [aws/aws-sdk-java#2933] [General]
We have a simple AmazonS3 client built using AmzonS3ClientBuilder configured to use MaxErrorRetry of 10. How does one technically validate this number using junit?

**Comment by debora-ito** [comment]
This is more a "how do I write tests for my application" question, not really a "how the SDK works" question.

Moving this to the General category hoping to gather suggestions from the community.
